Makefile

```diff
@@ -1,107 +1,113 @@
 #/bin/bash
 
-BASEDIR = $(shell pwd)
-SOURCE = "${BASEDIR}/lib"
-IMAGE = "elastica"
+.PHONY: update clean build setup start stop destroy run checkstyle checkstyle-ci code-browser cpd messdetector messdetector-ci dependencies phpunit test tests doc lint syntax-check loc phploc gource 
```

Comment:
A lot of targets, some of them just copypasted from ant config file that was deleted far ago :) 
Which of them is *actually* used?